src/panels/templates/AutomaticCreateCluster.json

```diff
@@ -53,9 +176,21 @@
                         "count": 3,
                         "osType": "Linux"
                     }
-                ]
+                ],
+                "supportPlan": "[parameters('supportPlan')]"
             },
             "identity": "[parameters('clusterIdentity')]"
+        },
+        {
+            "type": "Microsoft.Authorization/roleAssignments",
+            "apiVersion": "2020-04-01-preview",
+            "name": "[guid(resourceGroup().id)]",
+            "properties": {
+                "roleDefinitionId": "[concat('/subscriptions/', parameters('subscriptionId'), '/providers/Microsoft.Authorization/roleDefinitions/', 'b1ff04bb-8a4e-4dc4-8eb5-8693973ce19b')]",
```

Comment:
this is role definition ID of b1ff04bb-8a4e-4dc4-8eb5-8693973ce19b of Azure K8s Cluster Admin - https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles/containers#azure-kubernetes-service-rbac-cluster-admin